tasks/15_confirm-ending/confirm-ending.js

```diff
@@ -0,0 +1,7 @@
+export function confirmEnding(str, target) {
+    if(str.substr(-target.length) === target){
+        return true
```

Comment:
Returning `true` or `false` directly from function is a bad practice. You can return the result of equal operation